modules/mvt/src/lib/binary-vector-tile/vector-tile-layer.js

```diff
@@ -0,0 +1,79 @@
+// This code is forked from https://github.com/mapbox/vector-tile-js under BSD 3-clause license.
+
+import VectorTileFeature from './vector-tile-feature.js';
+
+export default class VectorTileLayer {
+  constructor(pbf, end) {
+    // Public
+    this.version = 1;
+    this.name = null;
+    this.extent = 4096;
```

Comment:
Can we have this as a parameter with default to 4096? Some backend providers allow to specify the tile extent

Reply:
Forget about this, It would be a nice feature, but it should be addressed in a separated PR